integration-tests/codestarts/src/test/resources/__snapshots__/VaadinExtensionPreReleaseCodestartTest/testApplicationContents/dir-tree.snapshot

```diff
@@ -0,0 +1,7 @@
+/
+themes/
+themes/starter-theme/
+themes/starter-theme/components/
+themes/starter-theme/components/vaadin-text-field.css
+themes/starter-theme/styles.css
+themes/starter-theme/theme.json
```

Comment:
smallie:
```suggestion
themes/starter-theme/theme.json

```

Reply:
Same as  before, this is file is generated by quarkus tooling, so better not modify it manually

Reply:
issue created:
- https://github.com/vaadin/quarkus/issues/120

Reply:
Thanks, makes sense in that case.